libs/repositories/src/repos/TokenBalancesRepository/TokenBalancesRepositoryMoralis.ts

```
@@ -0,0 +1,67 @@
+
+      const asJson = await response.json() as MoralisBalanceResponse
+      return asJson.result.reduce((acc, tokenBalanceItem) => {
+        acc[tokenBalanceItem.token_address.toLowerCase()] = tokenBalanceItem.balance
+        return acc
+      }, {} as Record<string, string>)
+    } catch (e) {
+      console.log(e)
+      return null
+    }
+
+    return null
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)